src/routing/RouteSelector.jsx

```diff
@@ -47,6 +48,12 @@ export const RouteSelector = () => (
     <Switch>
       <RedirectIfLoggedInRoute exact path="/" unauthedComponent={LoginPage} />
 
+      <RedirectIfLoggedInRoute
+        exact
+        path="/ogp-login"
```

Comment:
i'm not entirely sure that this is what was intended - do we have a rfc/spec to refer to? i think this endpoint might be exposed on the backend instead?

Reply:
This is intended, we have a separate login endpoint (`/ogp-login`) which ogp officers can use for now, so as not to confuse users about the availability of this functionality